Cargo.lock

```diff
@@ -2242,6 +2292,12 @@ dependencies = [
  "yoke",
 ]
 
+[[package]]
+name = "litemap"
+version = "0.7.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f9d642685b028806386b2b6e75685faadd3eb65a85fff7df711ce18446a422da"
```

Comment:
Issue: For some reason you're now linking a copy of `litemap` from Crates.io

Reply:
`md-tests`